src/vector_index.h

```diff
@@ -27,6 +30,7 @@ typedef struct VectorFilter {
   void *vector;                   // vector data
   size_t vecLen;                  // vector length
   VectorQueryType type;           // TOPK or RANGE
+  int isBase64;                   // uses base64 strings
```

Comment:
bool?

Reply:
Generally, we do not use bool in RediSearch. I guess it follows Redis OSS.

Reply:
But we do. Just search `bool` on the code base. I'm not familiar with such a convention.